inmap/cmd/config.go

```diff
@@ -118,8 +118,8 @@ func ReadConfigFile(filename string) (config *ConfigData, err error) {
 			"there has been an error parsing the configuration file: %v\n", err)
 	}
 
-	for i, v := range config.OutputVariables {
-		config.OutputVariables[i] = os.ExpandEnv(v)
+	for k, v := range config.OutputVariables {
+		config.OutputVariables[k] = os.ExpandEnv(v)
```

Comment:
It seemed like there was no error coming from this before, but maybe I just missed it. Why do we need to use os.ExpandEnv() at all? Does it have something to do with how TOML files are interpreted?